src/fundus/publishers/us/the_nation_parser.py

```diff
@@ -56,3 +66,13 @@ def title(self) -> Optional[str]:
         @attribute
         def topics(self) -> List[str]:
             return generic_topic_parsing(self.precomputed.meta.get("keywords"))
+
+    class V2(V1):
+        VALID_UNTIL = date.today()
+
+        _summary_selector = XPath("//article//div[contains(@class, 'article-title')]//p")
```

Comment:
Just very minor: The V1 (and other parsers) have direct typehints for these variables for their first definition.

Reply:
Hmm, you sure? I only type hinted this variable in V1 because otherwise mypy would complain that we overwrite the type (CSSSelector with XPath). IMO type hinting these is a bit overkill unless necessary.

Reply:
Yes, you are right. In most parsers, we did not do this. It was properly only the parsers I've worked on to indicate where the variable is defined first. If it's overkill for you that's fine.

Reply:
I forgot that `CSSSelect` is derived from `XPath` so I removed `Selector` and had to touch the types again and then I accidentally added some to the new selectors as well ;) I guess my fingers slipped.

Reply:
Haha, all good^^